tools/benchmarks/classification/knn_imagenet/slurm_test_knn_epoch.sh

```diff
@@ -32,5 +31,4 @@ srun -p ${PARTITION} \
     ${SRUN_ARGS} \
     python -u tools/benchmarks/classification/knn_imagenet/test_knn.py $CFG \
         --checkpoint $WORK_DIR/epoch_${EPOCH}.pth \
```

Comment:
better to set checkpoint path directly according to command line, not by epoch num

Reply:
The script style follows and is consistent with the `svm_voc07` benchmark.